### PR TITLE
Add config package to the requirements

### LINKED ISSUE #70: ModuleNotFoundError: No module named 'config'
**Describe the bug**
After installing rquirements and starting I get:
ModuleNotFoundError: No module named 'config'

**To Reproduce**
Just run pip3 -r requirements.txt on a machine that does not have all the python stuff a dev uses

**Additional context**
To make it short just add "config" to the requirements. Once added manually it works.

### PR DESCRIPTION
**Pull request by sundowndev**
Fixes #70

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,3 +4,4 @@ requests==2.20.0
 selenium==3.141.0
 beautifulsoup4==4.7.1
 html5lib==1.0.1
+config==0.4.2
```
Comment (alfaher) [inline requirements.txt]: hello, Im new to using this program, where can I find this document in my system?
kind regards

Comment (sundowndev) [reply requirements.txt]: @alfaher You might be looking at `ls -lha /usr/lib/python* | grep config`

**Comment by alfaher** [comment]
thank you, it worked!

**Comment by Bluucaat** [comment]
added config, but the same error keeps coming up to me

**Pull request merged**